common/src/main/java/net/william278/huskhomes/command/SavedPositionCommand.java

```diff
@@ -89,15 +89,42 @@ private Optional<Home> resolveHome(@NotNull CommandUser executor, @NotNull Strin
         return Optional.empty();
     }
 
-    private Optional<Home> resolveDelimitedHome(@NotNull CommandUser executor, @NotNull String homeName) {
-        final String ownerUsername = homeName.substring(0, homeName.indexOf(Home.getDelimiter()));
-        final String ownerHome = homeName.substring(homeName.indexOf(Home.getDelimiter()) + 1);
+    /**
+     * Parses a delimited home identifier of the form {@code owner + delimiter + homeName}.
+     * Uses the <b>last</b> delimiter occurrence to support owners whose names begin with the delimiter
+     * (e.g., Bedrock usernames like ".ciro123").
+     */
+    @NotNull
+    private static Optional<String[]> parseDelimitedHomeIdentifier(@NotNull String homeName) {
```

Comment:
With how simple this change effectively is, I'm not sure it warrants moving code into a function. The same result could likely be achieved in a cleaner way by moving the `lastIndexOf` to the top of `resolveDelimitedHome()` and modifying the following lines.